src/mono/browser/runtime/memory.ts

```diff
@@ -177,12 +186,17 @@ export function setF64(offset: MemOffset, value: number): void {
     Module.HEAPF64[<any>offset >>> 3] = value;
 }
 
-
+/** note: MonoBoolean is 8 bits not 32 bits when inside a structure or array */
 export function getB32(offset: MemOffset): boolean {
     receiveWorkerHeapViews();
     return !!(Module.HEAP32[<any>offset >>> 2]);
```

Comment:
as mentioned elsewhere i'd like to see this assert/warn if it encounters a value that isn't 0 or 1, but this is fine